### ISSUE TITLE
Assertion failure

### ISSUE THREAD

**Issue by agnesnatasya** [nus-cs2103-AY1920S2/forum#39]
When I run Duke from IntelliJ and I type in some commands in the UI, without going back to IntelliJ, everything works fine. 

But when I open IntelliJ when I was opening the UI (JavaFX UI), sometimes they give me this Assertion failure. 

<img width="892" alt="Screenshot 2020-02-11 at 1 22 36 PM" src="https://user-images.githubusercontent.com/44172431/74213050-49030800-4cd2-11ea-9ca0-abfb33ef7a84.png">

I tried to google it and some says that this is a bug in JavaFX. Does anyone knows that this is indeed a bug in JavaFX or I have some assertion failure inside my code?

Thank you!

**Comment by mario7lorenzo** [comment]
Hi! can you verify where you put `assert` in your code, and try comment them first? If the bug is still reproduced, I think it's your javafx version that has some issues. That's the way to figure out the issues when you don't have a clue of what to do.

**Comment by okkhoy** [comment]
@j-lum any idea about this?

**Comment by j-lum** [comment]
Looks to be a problem with JavaFX but it should have been fixed. @agnesnatasya can you check that you're on the right JDK/JavaFX?

**Comment by mario7lorenzo** [comment]
@j-lum Ah yeah, after doing more research, the JDK version could cause the problem as well.

**Comment by agnesnatasya** [comment]
Yes, I think it is a problem with JavaFX because the same error happens in the branch where there is no assertions. I am using Java 11 (when I type java --version in my terminal), and for the JavaFX I just follow the instruction in the tutorial :D

**Comment by j-lum** [comment]
Anyway the fixes appear to have been backport to openjfx 11. Try updating your gradle buildfile to pull in openjfx v11.0.6.

**Comment by agnesnatasya** [comment]
Okay, should I add these lines to my build.gradle?
`plugins {
    id 'org.openjfx.javafxplugin' version '0.0.7'
}

javafx {
    version = "11.0.6"
    modules = [ 'javafx.controls', 'javafx.fxml' ]
}`

But anyway, even though there is this error, my bot still runs and it does not stop :)

**Comment by agnesnatasya** [comment]
Hi, may I know if I could not resolved this issue, but my Duke still works fine, will this be okay for my iP submission? Thank you!
